src/cli/managers/config_manager.py

```diff
@@ -1,16 +1,47 @@
+import os
 import logging
 from pathlib import Path
-from typing import Any, Dict, List, Tuple
-
-from cli.utils import read_json_file
+from typing import Dict, List, Tuple, Any
+from cli.utils import read_json_file, write_json_file
 
 
 class ConfigManager:
     def __init__(self, config_dir: Path):
         self.__config_dir = config_dir
+        self.app_config_file = self.__config_dir / "applications.json"
 
     def add_application(self, application: str, path: Path) -> None:
-        pass
+        """
+        Takes care of saving the application name in the .qne/application.json root file together with the application
+        path.
+
+        Args:
+            application: the application name
+            roles: a list of roles
+            path: the path where the application is stored
+
+        """
+
+        # Read json file
+        apps = read_json_file(self.app_config_file)
+
+        # Store the app path
+        apps[application] = {'path': os.path.join(str(path), application) + "/"}
+        write_json_file(self.app_config_file, apps)
+
+    def check_config_exists(self) -> bool:
+        """ Checks if the application.json config file exists in the .qne/ root directory. Returns True when it does
+        exist, False if not.
+
+        return:
+        bool: True if 'app_config_file' exists, False if not
```

Comment:
The type of the return value must not be documented. It is clear from the code.
See Google style docstring (`https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html`)
specific:
`example function_with_pep484_type_annotations`